packages/react/src/patterns/sub-patterns/TableOfContents/TableOfContents.js

```diff
@@ -58,7 +58,7 @@ const TableOfContents = ({ menuItems, children, menuLabel, theme }) => {
      * @private
      * @returns {Array} returns elemenrt name and data title
      */
-    const _find_menu_items = () => {
+    const findMenuItems = () => {
```

Comment:
@carthick I also wouldn't define a method inside of a `useEffect`. It should be outside of it, but can be called from it.